src/migration/1658242193484-AddNewOneAITags.ts

```diff
@@ -0,0 +1,1387 @@
+            ('boss','deny'),
+            ('hashnode','deny'),
+            ('discovery','deny'),
+            ('blogging','deny'),
+            ('visa','deny'),
+            ('habits','deny'),
+            ('advantages','deny'),
+            ('data-driven','deny'),
+            ('messenger','deny'),
+            ('minimum-viable-product','deny'),
+            ('logistics','deny'),
+            ('scripting','deny'),
+            ('anonymous','deny'),
+            ('gm','deny'),
+            ('passion','deny'),
+            ('cto','deny'),
+            ('headphones','deny'),
+            ('pair-programming','deny'),
+            ('pixabay','deny'),
+            ('macbook-pro','deny'),
+            ('covid','deny'),
+            ('epic','deny'),
+            ('ea','deny'),
+            ('dataframes','deny'),
+            ('misinformation','deny'),
+            ('cisa','deny'),
+            ('content-marketing','deny'),
+            ('reuters','deny'),
+            ('makers','deny'),
+            ('hotel','deny'),
+            ('product-design','deny'),
+            ('dogs','deny'),
+            ('onboarding','deny'),
+            ('animals','deny'),
+            ('configuration-management','deny'),
+            ('fashion','deny'),
+            ('solarwinds','deny'),
+            ('tips-and-tricks','deny'),
+            ('frontend-development','deny'),
+            ('trading','deny'),
+            ('3d-printing','deny'),
+            ('drones','deny'),
+            ('ajax','deny'),
+            ('crud','deny'),
+            ('swap','deny'),
+            ('canonical','deny'),
+            ('coroutine','deny'),
+            ('shipping','deny'),
+            ('forecast','deny'),
+            ('gold','deny'),
+            ('economy','deny'),
+            ('scam','deny'),
+            ('next','deny'),
+            ('codecanyon','deny'),
+            ('doctors','deny'),
+            ('pc','deny'),
+            ('disney+','deny'),
+            ('emotions','deny'),
+            ('cover','deny'),
+            ('office-365','deny'),
+            ('ge','deny'),
+            ('oop','deny'),
+            ('polygon','deny'),
+            ('scopes','deny'),
+            ('android-app-development','deny'),
+            ('automotive','deny'),
+            ('ios-app-development','deny'),
+            ('anchor','deny'),
+            ('pexels','deny'),
+            ('gartner','deny'),
+            ('prototyping','deny'),
+            ('labor','deny'),
+            ('responsive-web-design','deny'),
+            ('labs','deny'),
+            ('dropbox','deny'),
+            ('clubhouse','deny'),
+            ('hashing','deny'),
+            ('dash','deny'),
+            ('congress','deny'),
+            ('electricity','deny'),
+            ('smashing','deny'),
+            ('lambda-function','deny'),
+            ('ui','deny') 
+            ON CONFLICT ("value") DO UPDATE SET status = EXCLUDED.status`,
+            );
+
+            await queryRunner.query(
```

Comment:
I think this list should be `allow` not `pending`

Reply:
Made changes